src/directives/view.js

```diff
@@ -257,8 +259,27 @@ class routeViewDirective {
         if (component.controller) {
           const locals = _.merge(dependencies, {$scope: viewScope, $element: element.children().eq(0)});
 
-          const controller = $controller(component.controller, locals);
-          locals.$scope[component.controllerAs] = controller;
+          try {
```

Comment:
the assignment is unnecessary (i.e., controller is not referenced anywhere else)?